.dockeringore

```diff
@@ -0,0 +1,3 @@
+.venv
+result.png
+Dockerfile
```

Comment:
I think you may add some more items here

Reply:
You definitely need to add more files.